docs/reference/migration/migrate_7_0/java.asciidoc

```diff
@@ -5,4 +5,10 @@
 
 `isShardsAcked` has been replaced by `isShardsAcknowledged` in
 `CreateIndexResponse`, `RolloverResponse` and
-`CreateIndexClusterStateUpdateResponse`.
\ No newline at end of file
+`CreateIndexClusterStateUpdateResponse`.
+
+==== `prepareExecute` removed from the client api
+
+The `prepareExecute` method which created a request builder has been
+removed from the client api. Instead, construct a build for the
```

Comment:
not sure if this should be "builder" here

Reply:
The prepareExecute method returned a RequestBuilder. Is there something else you think it should be?

Reply:
Oops, I see what you mean, thanks!